lib/SIL/SILProfiler.cpp

```diff
@@ -151,8 +152,10 @@ SILProfiler *SILProfiler::create(SILModule &M, ForDefinition_t forDefinition,
   if (!doesASTRequireProfiling(M, N) && Opts.UseProfile.empty())
     return nullptr;
 
-  if (!canCreateProfilerForAST(N, forDecl))
+  if (!canCreateProfilerForAST(N, forDecl)) {
+    N.dump(llvm::errs());
```

Comment:
Left this in by accident, but I'm not convinced it's actually a bad idea.

Reply:
It seems helpful to me.